packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -0,0 +1,64 @@
+
+## User Interactions
+
+The React Scheduler provide a functionality to manage the data source via user interface. End-user can add, edit and delete appointments by built-in UI controls. Rules below introduce to user's editing manipulations.
+
+### Add Appointment
+
+1. Double clicks a cell in a timetable. The appointment editing form will be shown.
+2. In the form, specifies required fields and clicks the `Create` button. This will create an appointment and add it.
+
+### Update Appointment
+
+- Double clicks an appointment. The appointment editing form will be shown.
+- Clicks an appointment. The appointment tooltip will be shown with an `Edit` button. The `Edit` button will open an appointment editing form.
+
+### Delete Appointment
+
+- Clicks an appointment. The appointment tooltip will be shown with a `Delete` button.
+
+## Related Plugins
+
+The following plugins implement editing features:
+
+- [EditingState](../reference/editing-state.md) - controls the editing state
+- [AppointmentTooltip](../reference/appointment-tooltip.md) - renders a appointment tooltip with editing controls
+- [AppointmentForm](../reference/appointment-form.md) - renders an appointment editing form
+- [Appointments](../reference/appointments.md) - renders appointments
+
+## Basic Setup
+
+Add the plugins listed above to the Scheduler to set up a simple Scheduler supporting editing features.
+
+Handle the `EditingState` plugin's `onCommitChanges` event to commit changes made by an end-user to your data store.
+
+## Uncontrolled Mode
+
+In the [uncontrolled mode](controlled-and-uncontrolled-modes.md), you can specify the initial editing state values using the following `EditingState` plugin's properties:
+
+- `defaultEditingAppointmentId` - the appointment ID being edited
+- `defaultAddedAppointment` - the appointment being added
+- `defaultAppointmentChanges` - the appointment changes
+
+.embedded-demo({ "path": "scheduler-editing/uncontrolled", "showThemeSelector": true })
+
+## Controlled Mode
+
+In the [controlled mode](controlled-and-uncontrolled-modes.md), specify the following `EditingState` plugin's property pairs to set a state value and handle its changes:
+
+- `editingAppointmentId` and `onEditingAppointmentIdChange` - the appointment ID being edited
+- `addedAppointment` and `onAddedAppointmentChange` - the appointment being added
+- `appointmentChanges` and `onAppointmentChangesChange` - the appointment changes
+
+Note, you can also use the `onAddedAppointmentChange` event to initialize a created appointment with default property values.
+
+.embedded-demo({ "path": "scheduler-editing/controlled", "showThemeSelector": true })
+
+## Delete Confirmation
```

Comment:
I'm not sure it's a good name for this section. Actually, you can prevent or confirm any user action: add, update and remove. It can look like:

## Changes preprocessing

You can validate user input or prevent user action using the `onCommintChanges` property of the `EditingState` plugin. For example, it's a quite popular task to show a confirmation dialog to prevent accidental data removing. Let's see how it can be implemented......